README.md

```diff
@@ -1,169 +1,33 @@
-
-```bash
-gcloud auth login
-gcloud auth application-default login
-```
-
-## Run
-
-This repository contains example ML training and prediction pipelines which are explained in [this guide](docs/Pipelines.md).
-
-**Build containers:** The [model/](/model/) directory contains the code for custom training and prediction container images, including the model training script at [model/training/train.py](model/training/train.py). 
-You can modify this to suit your own use case.
-Build the training and prediction container images and push them to Artifact Registry with:
-
-```bash
-make build [ images="training prediction" ]
-```
-
-Optionally specify the `images` variable to only build one of the images.
-
-**Execute pipelines:** Vertex AI Pipelines uses KubeFlow to orchestrate your training steps, as such you'll need to:
-
-1. Compile the pipeline
-1. Build dependent Docker containers
-1. Run the pipeline in Vertex AI
-
-Execute the following command to run through steps 1-3:
-
-```bash
-make run pipeline=training [ build=<true|false> ] [ compile=<true|false> ] [ cache=<true|false> ] [ wait=<true|false> ] 
-```
-
-The command has the following true/false flags:
-
-- `build` - re-build containers for training & prediction code (limit by setting images=training to build only one of the containers)
-- `compile` - re-compile the pipeline to YAML
-- `cache` - cache pipeline steps
-- `wait` - run the pipeline (a-)sync
-
-**Shortcuts:** Use these commands which support the same options as `run` to run the training or prediction pipeline:
-
-```bash
-make training
-make prediction
-```
-
-## Test
-
-Unit tests are performed using [pytest](https://docs.pytest.org).
-The unit tests are run on each pull request. 
-To run them locally you can execute the following command and optionally enable or disable testing of components:
-
-```
-make test [ packages=<pipelines components> ]
-```
-
-## Automation
-
-For details on setting up CI/CD, see [this guide](./docs/Automation.md).
-
-## Putting it all together
-
-For a full walkthrough of the journey from changing the ML pipeline code to having it scheduled and running in production, please see the guide [here](./docs/Production.md).
-
-We value your contribution, see [this guide](./docs/Contribution.md) for contributing to this project.
+1. Create 3x Google Cloud projects (dev, test, prod)
+1. Use `make deploy` to deploy resources in each of them. It's advised to follow the [infrastructure setup notebook](./docs/notebooks/01_infrastructure_setup.ipynb) for each environment
+1. Create an E2E test trigger in the test project
+1. Create a release trigger in the prod project
+1. Add each user with their own Google account with the following IAM roles:
+    - `Vertex AI User` (roles/aiplatform.user)
+    - `Storage Object Viewer` (roles/storage.objectViewer)
+    - `Service Usage Consumer` (roles/serviceusage.serviceUsageConsumer)
+1. Create one workbench instance (Google Colab Enterprise is recommended) per user.
```

Comment:
Tried cloning into Google Colab and opening the notebooks but they don't open very nicely. All the text is in one big cell. Maybe we should stick to Workbench instances